pkg/pdapi/pdapi.go

```diff
@@ -697,6 +718,7 @@ const (
 	DeleteMemberByIDActionType         ActionType = "DeleteMemberByID"
 	DeleteMemberActionType             ActionType = "DeleteMember "
 	SetStoreLabelsActionType           ActionType = "SetStoreLabels"
+	UpdatePlacementRulesActionType     ActionType = "UpdateEnablePlacementRules"
```

Comment:
I think PlacementRules is part of the `ReplicationConfig`, We should name it as `SetReplicationConfig`

Reply:
There are other fields in this struct, here I only update this one field, if need to update other fields, we should define another function, they may be used in different scenarios.

Reply:
updated.